Aardvark/ARKLogType.h

```diff
@@ -0,0 +1,32 @@
+//
+//  ARKLogType.h
+//  Aardvark
+//
+//  Created by Evan Kimia on 10/22/15.
+//  Copyright © 2015 Square, Inc. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+@import UIKit;
```

Comment:
Let's use `#import <UIKit/UIKit.h>` here again. Mixed `@import` and `#import` can lead to some oddness that I'd prefer to avoid for now.